libs/blockscout-chains/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "blockscout-chains"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
Do we make use of that somewhere already? It is probably better to bump it to "0.2.0", as otherwise it may be updated automatically and cause compilation errors

Reply:
Yes, we use it in metadata service already. Updated to `0.2.0`